packages/test/test-end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -287,7 +287,7 @@ describeFullCompat("Detached Container", (getTestObjectProvider) => {
         await defPromise.promise;
     });
 
-    it("Fire dataStore attach ops during container attach", async () => {
+    it.skip("Fire dataStore attach ops during container attach", async () => {
```

Comment:
Broken Test: detachedContainerTests.spec.ts: Fire dataStore attach ops during container attach

The test's assert was never handled correctly, so likely has always thrown

Reply:
#9040